tests/aitchison-tests.R

```diff
@@ -34,6 +35,10 @@ a1 <- vegan::vegdist(testdata, method = "aitchison", pseudocount=1)
 a2 <- vegan::vegdist(testdata+1, method = "aitchison")
 max(abs(a1-a2)) < 1e-6 # Tolerance
 
+# Robust rclr+Euclid should give same result than robust.aitchison
+d1 <- vegan::vegdist(vegan::decostand(testdata, "rclr"), "euclidean")
```

Comment:
In vegan, do not use vegan::function(): Ripley can catch this and complain (that has happened to me).